src/stringProcessing/checkNofollow.js

```diff
@@ -1,17 +1,38 @@
 /** @module stringProcessing/checkNofollow */
 
+// We use an external library, which can be found here: https://github.com/fb55/htmlparser2.
+let htmlparser = require( "htmlparser2" );
+let isString = require( "lodash/isString" );
+
 /**
  * Checks if a links has a nofollow attribute. If it has, returns Nofollow, otherwise Dofollow.
  *
  * @param {string} text The text to check against.
  * @returns {string} Returns Dofollow or Nofollow.
  */
 module.exports = function( text ) {
-	var linkFollow = "Dofollow";
+	let linkFollow = "Dofollow";
+
+	let parser = new htmlparser.Parser( {
+		/**
+		 * Detects if there is a `nofollow` argument in the `rel` attribute of a link.
+		 *
+		 * @param {string} tagName The tag name.
+		 * @param {object} nodeValue The attribute with the keys and values of the tag.
```

Comment:
I guess this is copied from `src/stringProcessing/htmlParser.js` however, this object actually contains _all_ the attributes found in a tag. I'd consider to rename `nodeValue` to something more meaningful and improve hte description.

Reply:
It actually contains all attributes in this case as well, though we are only interested in the `rel` attribute in processing.